content/api/commands/intercept.md

```diff
@@ -293,26 +295,29 @@ cy.wait('@searchForTerms')
 ```js
 cy.intercept(
   {
-    // this RegExp matches any URL beginning with 'http://api.example.com/' and ending with '/edit' or '/save'
+    // this RegExp matches any URL beginning with
+    // 'http://api.example.com/' and ending with '/edit' or '/save'
     url: /^http:\/\/api\.example\.com\/.*\/(edit|save)/,
     headers: {
       'x-requested-with': 'exampleClient',
     },
   },
   (req) => {
-    // only requests to URLs starting with 'http://api.example.com/widgets'
-    // and having the header 'x-requested-with: exampleClient' will be intercepted
+    // only requests to URLs starting with
+    // 'http://api.example.com/widgets' and having the header
+    // 'x-requested-with: exampleClient' will be intercepted
   }
 })
 
-// in this example, the supplied URL `/users` is merged with the RouteMatcher
-// passed as the second argument
+// in this example, the supplied URL `/users` is merged with
+// the RouteMatcher passed as the second argument
 cy.intercept('/users', { middleware: true }, (req) => {
   req.headers['authorization'] = `Bearer ${bearerToken}`
 })
 
-// this example will cause 1 request to `/temporary-error` to receive a network error
-// and subsequent requests will not match this `RouteMatcher`
+// this example will cause 1 request to `/temporary-error`
+// to receive a network error and subsequent requests will
+// not match this `RouteMatcher`
 cy.intercept('/temporary-error', { times: 1 }, { forceNetworkError: true })
```

Comment:
Can this line of code be wrapped so that the example doesn't require horizontal scrolling?

Reply:
It's not worth it IMO

Reply:
This PR is addressing the concern that code comments aren't fully readable on a standard size screen without scrolling.

I know the original observation was about code comments, but I think we should also apply the same logic to all code in code examples in order to minimize (or eliminate) horizontal scrolling, to maximize readability.

Reply:
Make a code suggestion that you feel maximizes readability.